cairo-libs/packages/cometbft/tests/verifier.cairo

```diff
@@ -0,0 +1,85 @@
+use cometbft::light_client::Header as LcHeader;
+use cometbft::types::{Header as TmHeader, Options, TrustedBlockState, UntrustedBlockState};
+use cometbft::utils::TWO_THIRDS;
+use cometbft::verifier::{header_matches_commit, verify_update_header};
+use protobuf::types::message::ProtoCodecImpl;
+use protobuf::types::wkt::{Duration, Timestamp};
+use protobuf::{base64, hex};
+
+#[test]
+fn test_verify_update_header() {
```

Comment:
Managed to get tendermint light client verified !

This test called the top `verify_update_header` 🙂

The header data is captured directly from hermes-sdk update client payload.